.ci/generate_bear_requirements.py

```diff
@@ -155,9 +156,11 @@ def write_npm_requirements(requirements):
 
     package_json_string = template.render(
         dependencies=npm_dependencies, version="0.8.0")
+    pretty_json = json.dumps(json.loads(package_json_string, object_pairs_hook=OrderedDict), indent=2)
```

Comment:
Nice.
Could you move `indent=2` onto the next line, to make it a bit easier to read what is happening here.